src/styles/card.css

```
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  max-width: var(--card-max-width);
+  background-color: var( --card-background-color);
+  border: var(--card-border) solid var(--border-color);
+  border-radius: var(--card-border-radius);
+}
+
+.card__header {
+  width: 100%;
+  margin: var(--card-margin);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 19px;
+  padding-bottom: 0;
+  padding-top: var(--card-header-top-gap);
```

```suggestion
  padding: var(--card-header-top-gap) 19px 0
```
You can just type it like that. Please fix all cases.
Btw, there is no need to create variables for each value. Add them only to the main values